traitsui/qt4/ui_base.py

```diff
@@ -556,45 +595,6 @@ def set_status_text(text):
 
         return set_status_text
 
-    #---------------------------------------------------------------------------
-    #  Adds a menu item to the menu bar being constructed:
-    #---------------------------------------------------------------------------
-
-    def add_to_menu ( self, menu_item ):
-        """ Adds a menu item to the menu bar being constructed.
-        """
-        item   = menu_item.item
-        action = item.action
-
-        if action.id != '':
-            self.ui.info.bind( action.id, menu_item )
-
-        if action.style == 'radio':
-            if ((self._last_group is None) or
-                (self._last_parent is not item.parent)):
-                self._last_group = RadioGroup()
-                self._last_parent = item.parent
-            self._last_group.items.append( menu_item )
-            menu_item.group = self._last_group
-
-        if action.visible_when != '':
-            self.ui.add_visible( action.visible_when, menu_item )
-
-        if action.enabled_when != '':
-            self.ui.add_enabled( action.enabled_when, menu_item )
-
-        if action.checked_when != '':
-            self.ui.add_checked( action.checked_when, menu_item )
-
-    #---------------------------------------------------------------------------
-    #  Adds a tool bar item to the tool bar being constructed:
-    #---------------------------------------------------------------------------
-
-    def add_to_toolbar ( self, toolbar_item ):
-        """ Adds a toolbar item to the toolbar being constructed.
-        """
-        self.add_to_menu( toolbar_item )
-
     def can_add_to_menu(self, action, action_event=None):
```

Comment:
These functions were in `BaseDialog` https://github.com/enthought/traitsui/blob/master/traitsui/qt4/ui_base.py#L563, not in `_StickyDialog`.  diff is confused.
`BaseDialog` inherits `BasePanel`.